### PR TITLE
Fix camera issue on Android 12

### LINKED ISSUE #1904: Android 12 and blank white screen for taking photos
When I try to take a photo the camera appears with a blank white screen.

### PR DESCRIPTION
**Pull request by tomasMizera**
Android introduced property `queries` in manifest that should contain information about our use of external applications. It is called `Package visibility`. As we did not declare it, they disabled our access to device's camera.

In order to use the new `queries` keyword **we needed to update Gradle version** to new point release that Google released for [this use case](https://android-developers.googleblog.com/2020/07/preparing-your-build-for-package-visibility-in-android-11.html).

In the end the issue had nothing to do with `HIGH_SAMPLING_RATE_SENSORS` permission. However, on Android 12, sensor `TYPE_MAGNETIC_FIELD` (we use it in `OrientationSensor.java` to determine orientation when taking a photo) does not allow us to read data like we read it now (ASAP). 

_One more thing to note: Updating Gradle creates a mismatch between our and Qt 5.14.2's Gradle version. It is a difference only in point release_ 

Read more:
https://developer.android.com/training/package-visibility
https://developer.android.com/training/package-visibility/declaring
https://android-developers.googleblog.com/2020/07/preparing-your-build-for-package-visibility-in-android-11.html
https://stackoverflow.com/questions/67455534/how-to-add-queries-declaration-in-the-manifest-for-using-camera

Fixes #1904

### CODE REVIEW

**Comment by saberraz** [comment]
Excellent work. It behaves as expected on my pixel.

File: app/android/AndroidManifest.xml [hunk 1]
```diff
@@ -110,11 +110,22 @@
     <!-- Camera permissions -->
     <uses-permission android:name="android.permission.CAMERA"/>
 
+    <!-- In order to target Android 12 (API LVL 31), uncomment this permission, it is used for orientation sensor! -->
```
Comment (PeterPetrik) [inline app/android/AndroidManifest.xml:113]: is this sentence correct?

**Review by PeterPetrik — APPROVED** [review]
good work!

File: app/android/AndroidManifest.xml (hunk 1 above)
Comment (tomasMizera) [reply app/android/AndroidManifest.xml]: Changed that

**Pull request merged**